connector/github/github.go

```diff
@@ -260,12 +264,17 @@ func (c *githubConnector) HandleCallback(s connector.Scopes, r *http.Request) (i
 	if username == "" {
 		username = user.Login
 	}
+
 	identity = connector.Identity{
 		UserID:        strconv.Itoa(user.ID),
 		Username:      username,
 		Email:         user.Email,
 		EmailVerified: true,
 	}
+	if c.useLoginAsId {
+		identity.UserID = user.Login
+	}
+
```

Comment:
Travis fails because of this extra newline, I believe.

Reply:
removed the newline